src/index.js

```diff
@@ -25,7 +25,12 @@ class RetryChunkLoadPlugin {
           : '"cache-bust=true"';
       mainTemplate.hooks.localVars.tap(
         { name: pluginName, stage: 1 },
-        (source) => {
+        (source, chunk) => {
+          const currentChunkName = chunk.name;
+          const addRetryCode =
+            this.options.chunks == null ||
```

Comment:
The linter is failing because of this, I think it's better to keep it as `!this.options.chunks` as it will be `undefined` and not `null` if the option isn't passed